src/System.IO.FileSystem/tests/PortedCommon/IOInputs.cs

```diff
@@ -21,7 +21,11 @@ internal static class IOInputs
     // Unix values vary system to system; just using really long values here likely to be more than on the average system
     public static readonly int MaxDirectory = 247; // Does not include trailing \0. This the maximum length that can be passed to APIs taking directory names, such as Directory.CreateDirectory, Directory.Move
     public static readonly int MaxPath = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? 259 : 10000;      // Does not include trailing \0.
-    public static readonly int MaxComponent = 255;
+    public static readonly int MaxExtendedPath = short.MaxValue - 1;
```

Comment:
The -1 is because of trailing \0 right? I'd suggest adding a comment like above that highlights this.